src/pages/RestrictedAction/Workspace/WorkspaceOwnerRestrictedAction.tsx

```diff
@@ -20,6 +20,7 @@ function WorkspaceOwnerRestrictedAction() {
     const styles = useThemeStyles();
 
     const addPaymentCard = () => {
+        Navigation.resetToHome();
```

Comment:
I think we can use `Navigation.closeRHPFlow();` instead to pop the restricted action view from nav stack.